WireDomain/Sources/WireDomain/Notifications/Builders/Conversation/ConversationCallingEventNotificationBuilder.swift

```diff
@@ -37,16 +37,25 @@ struct ConversationCallingEventNotificationBuilder: ConversationCallingEventNoti
             return nil
         }
 
+        var resolvedConversationID: ConversationID {
```

Comment:
wouldn't it be better to avoid using a computed property here ? `resolvedConversationID` is called 3 times below, maybe it's better to set it once ? what do you think ?